extensions/interactions/NumericInput/directives/numeric-input-validation.service.ts

```diff
@@ -126,7 +126,7 @@ export class NumericInputValidationService {
           case 'Equals':
             var x = (<number>rule.inputs.x);
             setLowerAndUpperBounds(range, x, x, true, true);
-            raiseWarningForinputGreaterThanOrEqualToZero(j, x);
+            raiseWarningForrequireNonnegativeInput(j, x);
```

Comment:
Here and elsewhere: please proofread your copy/paste. Where "require" isn't the start of the variable, it should be capitalized in accordance with lowerCamelCase. I've seen this in other places too so please proofread your PR carefully.

Reply:
Done PTAL.